src/config/axios.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const clienteAxios = axios.create({
+    baseURL:'https://api.themoviedb.org/3/'
+});
+
+export default clienteAxios;
```

Comment:
Always include an end of file line break. I'll expand on this in a general comment.